src/Trading/components/TradingForm.tsx

```diff
@@ -94,10 +99,18 @@ function TradingForm(props: Props) {
 
   const horizon = useHorizon(props.account.testnet)
   const tradePair = useLiveOrderbook(primaryAsset || Asset.native(), secondaryAsset, props.account.testnet)
+  const openOrders = useLiveAccountOffers(props.account.publicKey, props.account.testnet)
 
   const assets = React.useMemo(() => props.trustlines.map(balancelineToAsset), [props.trustlines])
 
-  const calculation = useCalculation(form.getValues(), tradePair, priceMode, props.accountData, props.primaryAction)
+  const calculation = useCalculation(
```

Comment:
Now we are reaching the point where we have so many params that we should pass an options object and not single arguments anymore, so we can name them.

I can live with it for now, though. Just as a rule of thumb I think it's fair to say that starting with maybe 5 parameters you should consider an object and having six parameters you pretty much have to refactor if you don't wanna go crazy 😉